app/redux/services/getAllTxosForAccount.ts

```diff
@@ -0,0 +1,18 @@
+import * as fullServiceApi from '../../fullService/api';
+import { StringHex } from '../../types';
+import { errorToString } from '../../utils/errorHandler';
+import { getAllTxosForAccountAction } from '../actions';
+import { store } from '../store';
+
+export const getAllTxosForAccount = async (accountId: StringHex): Promise<void> => {
+  try {
+    const txos = await fullServiceApi.getAllTxosForAccount({ accountId });
+
+    if (store.getState().txos !== txos) {
```

Comment:
I don't think this conditional would ever result in `true`. If you want to compare object value equality you need to use something like lodash `isEqual`

Reply:
TIL! According to the MDN, `===` does a value and type comparison, but nothing is said about reference comparisons. Checking this out in JSfiddle though tells a different story and backs your comment up: https://jsfiddle.net/Laoxbudm/1/

Reply:
we can just use the `sameObject` util for this though, rather than adding in the lodash library just for this. It makes use of comparisons between `JSON.stringify(object)`s, so should handle this nicely. String literals should get compared correctly with `===`